src/kixi/nhs/board_report.clj

```diff
@@ -116,7 +222,7 @@
          data        (storage/get-resource-data ckan-client resource_id)]
     (patient-experience-of-gp-services recipe data)))
 
-(defn process-patient-experience-recipes [ckan-client recipes]
+(defn patient-experience-ethnicity-analysis [ckan-client recipes]
```

Comment:
How about this then:

```clojure
(defn patient-experience-deprivation-analysis [ckan-client {:keys []enthicity}]
  (mapcat #(process-deprivation-analysis ckan-client %) recipes))
```

and the same for the other similar function, just so that we know here that we're getting a filtered list?